packages/blob/src/del.ts

```diff
@@ -20,17 +12,13 @@ export async function del(
   url: string[] | string,
   options?: BlobCommandOptions,
 ): Promise<void> {
-  const blobApiResponse = await fetch(getApiUrl('/delete'), {
-    method: 'POST',
-    headers: {
-      ...getApiVersionHeader(),
-      authorization: `Bearer ${getTokenFromOptionsOrEnv(options)}`,
-      'content-type': 'application/json',
+  await requestApi(
+    '/delete',
+    {
+      method: 'POST',
+      headers: { 'content-type': 'application/json' },
```

Comment:
I guess we could pass content-type: 'application/json' all the time without any issues or problem, thoughts?

Reply:
i think actually only `del` and `multipart-complete` **need** `application/json`